readme.md

```diff
@@ -245,6 +245,17 @@ Returns `true` if the value is a `string` and the `.length` is 0.
 
 Returns `true` if the value is a `string` and the `.length` is more than 0.
 
+##### .nonEmptyStringAndNotWhitespace(value)
+
+Returns `true` if `is.string(value)` and if it's a `string` that is not a whitespace.
```

Comment:
This description needs work. "not a whitespace" is correct. And it's also missing "non-empty".

Reply:
updated